models/User.js

```diff
@@ -41,6 +41,7 @@ UserSchema.pre('save', async function () {
 });
 
 UserSchema.methods.comparePassword = async function (candidatePassword) {
+  // AKOSREVIEW: could just return the expression directly
   const isMatch = await bcrypt.compare(candidatePassword, this.password);
   return isMatch; //true
```

Comment:
This could be simplified a bit by just returning the value instead of saving it to a variable first:

`return await bcrypt.compare(candidatePassword, this.password);`

Reply:
UserSchema.methods.comparePassword = async function (candidatePassword) {
  return await bcrypt.compare(candidatePassword, this.password); //returns true or false
};